### PR TITLE
chore: add github issue / pr template

### PR DESCRIPTION
**Pull request by mhevery**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+**Note: for support questions, please use one of these channels:** https://github.com/angular/angular.js/blob/master/CONTRIBUTING.md#question. This repository's issues are reserved for feature requests and bug reports.
```
Comment (alexeagle) [inline .github/ISSUE_TEMPLATE.md]: update angular.js here?

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+* **Do you want to request a *feature* or report a *bug*?**
+
+
+
+* **What is the current behavior?**
+
+
+
+* **If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem** via
+https://plnkr.co or similar (you can use this template as a starting point: http://plnkr.co/edit/tpl:yBpEi4).
```
Comment (alexeagle) [inline .github/ISSUE_TEMPLATE.md]: wrong plnkr link

File: .github/ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+* **What is the motivation / use case for changing the behavior?**
+
+
+
+* **Which version of Angular, and which browser and OS does this issue affect?** Did this work in previous
+versions of Angular? Please also test with the latest stable and snapshot versions.
+
+
+
+* **Other information** (e.g. detailed explanation, stacktraces, related issues, suggestions how to fix)
```
Comment (alexeagle) [inline .github/ISSUE_TEMPLATE.md]: add "links for us to have context, eg. stackoverflow, gitter, etc"

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+* **What is the new behavior (if this is a feature change)?**
+
+
+
+* **Does this PR introduce a breaking change?**
+
+
+
+* **Please check if the PR fulfills these requirements**
```
Comment (alexeagle) [inline .github/PULL_REQUEST_TEMPLATE.md]: I'd put this first

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
+* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
```
Comment (alexeagle) [inline .github/PULL_REQUEST_TEMPLATE.md]: if the commit message is properly updated, then this is redundant, right?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+
+* **What is the current behavior?** (You can also link to an open issue here)
+
+
+
+* **What is the new behavior (if this is a feature change)?**
+
+
+
+* **Does this PR introduce a breaking change?**
```
Comment (alexeagle) [inline .github/PULL_REQUEST_TEMPLATE.md]: should we clarify what is a breaking change? this is not totally clear.
Maybe instead "What changes might users need to make in their application due to this PR?"

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (mhevery) [inline .github/ISSUE_TEMPLATE.md]: fixed

File: .github/ISSUE_TEMPLATE.md [hunk 7]
```diff
@@ -0,0 +1,29 @@
+**Note: for support questions, please use one of these channels:** https://github.com/angular/angular.js/blob/master/CONTRIBUTING.md#question. This repository's issues are reserved for feature requests and bug reports.
+
+* **Do you want to request a *feature* or report a *bug*?**
```
Comment (IgorMinar) [inline .github/ISSUE_TEMPLATE.md]: how about turning this into an ASCI checkboxes?

```
I'm submitting a ...
[ ] bug report
[ ] feature request
[ ] support request => Please do not submit support request here, see note at the top of this template.
```

File: .github/ISSUE_TEMPLATE.md (hunk 2 above)
Comment (mhevery) [inline .github/ISSUE_TEMPLATE.md]: nice catch

File: .github/ISSUE_TEMPLATE.md (hunk 3 above)
Comment (mhevery) [inline .github/ISSUE_TEMPLATE.md]: added

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 5 above)
Comment (mhevery) [inline .github/PULL_REQUEST_TEMPLATE.md]: correct

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 4 above)
Comment (mhevery) [inline .github/PULL_REQUEST_TEMPLATE.md]: done

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 6 above)
Comment (mhevery) [inline .github/PULL_REQUEST_TEMPLATE.md]: added

File: .github/ISSUE_TEMPLATE.md (hunk 7 above)
Comment (mhevery) [inline .github/ISSUE_TEMPLATE.md]: done

File: .github/ISSUE_TEMPLATE.md [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+* **What is the expected behavior?**
+
+
+
+* **What is the motivation / use case for changing the behavior?**
+
+
+
+* **Which version of Angular, and which browser and OS does this issue affect?** Did this work in previous
+versions of Angular? Please also test with the latest stable and snapshot versions.
```
Comment (IgorMinar) [inline .github/ISSUE_TEMPLATE.md]: Please tell us about your environment:
- Angular version: 2.0.0-beta.X
- Browser: [all | Chrome XX | Firefox XX | IE XX | Safari XX | Mobile Chrome XX | Android X.X Web Browser | iOS XX Safari | iOS XX UIWebView | iOS XX WKWebView ]
- Language: [all | TypeScript X.X | ES6/7 | ES5 | Dart]

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 5 above)
Comment (IgorMinar) [inline .github/PULL_REQUEST_TEMPLATE.md]: ASCI checkboxes?

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 6 above)
Comment (IgorMinar) [inline .github/PULL_REQUEST_TEMPLATE.md]: chebboxes

File: .github/ISSUE_TEMPLATE.md (hunk 8 above)
Comment (mhevery) [inline .github/ISSUE_TEMPLATE.md]: done

**Comment by vikerman** [comment]
Manually merged to master.

**Pull request closed without merging**